test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -86,9 +85,14 @@ void TestOutputHelper::printTestExecStats()
 {
 	if (Options::get().exectimelog)
 	{
-		std::cout << std::left;
+        int totalTime = 0;
+        std::cout << std::left;
```

Comment:
The stream flags should be restored in the end of the function.

Reply:
@winsvega maybe output to `strstream` and then the whole string to `cout` to avoid dealing with restoring `cout` state

Reply:
You can use `boost::io::ios_flags_saver` for it.